src/store/modules/global/saga.ts

```
@@ -306,10 +314,7 @@ export function* getAllPregnancyRecords(action: PayloadAction<{ userId: string }
     if (res.success) {
       yield put(setPregnancyRecord(res.response))
     }
-  } catch (error: any) {
-    message.error('An unexpected error occurred try again later!')
-    console.error('Fetch error:', error)
-  }
+  } catch (error: any) {}
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)